desktop/packages/linux/rs.ruffle.Ruffle.desktop.in

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Ruffle
+GenericName=Flash Player
+Comment=Play Flash games & movies
+Icon=rs.ruffle.Ruffle
+Exec=ruffle %u
+MimeType=application/x-shockwave-flash;application/vnd.adobe.flash.movie
+Categories=AudioVideo;Player;Graphics;Viewer;VectorGraphics;Game
+Keywords=flash;swf;player;emulator;rust
```

Comment:
Is the fact that this value is also offered for translation pre-determined by some of the tools used?

Reply:
Yes, keywords are translatable in general, the question is whether all of them should be. By browsing Wikipedia I saw that all of them could be translated, or at least transliterated.